HISTORY.rst

```diff
@@ -2,6 +2,16 @@
 History
 =======
 
+UNRELEASED
+----------
```

Comment:
I have a release script that inserts the version comment, the "UNRELEASED" header will get in the way of that 😉
```suggestion
UNRELEASED
----------

```